sentry-ruby/lib/sentry/utils/http_tracing.rb

```diff
@@ -55,6 +55,14 @@ def build_nested_query(value, prefix = nil)
           "#{URI.encode_www_form_component(prefix)}=#{URI.encode_www_form_component(value)}"
         end
       end
+
+      private
+
+      def get_level(status)
+        return :info unless status && status.is_a?(Integer)
```

Comment:
Is it really OK to default to :info when we don't recognize the status? Shouldn't this `log_warn("Unexpected HTTP status for breadcrumb: #{status.inspect}")`?

Reply:
yes
https://github.com/getsentry/relay/blob/bc70b9e3401bccb69e362acf16b9e6bf630aa3a2/relay-event-schema/src/protocol/breadcrumb.rs#L90-L95